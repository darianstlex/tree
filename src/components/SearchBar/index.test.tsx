import React from 'react';
import renderer from 'react-test-renderer';
import { SearchBar } from './index';

const onSearch = jest.fn();

describe('SearchBar', () => {
  it('renders properly', () => {
    const tree = renderer
      .create(<SearchBar onSearch={onSearch}/>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
})
