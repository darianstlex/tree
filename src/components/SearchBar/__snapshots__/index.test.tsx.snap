// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchBar renders properly 1`] = `
<span
  className="ant-input-group-wrapper ant-input-group-wrapper-lg ant-input-search ant-input-search-large ant-input-search-with-button search"
>
  <span
    className="ant-input-wrapper ant-input-group"
  >
    <span
      className="ant-input-affix-wrapper ant-input-affix-wrapper-lg"
      onMouseUp={[Function]}
      style={null}
    >
      <input
        className="ant-input ant-input-lg"
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        placeholder="git search string"
        style={null}
        type="text"
        value=""
      />
      <span
        className="ant-input-suffix"
      >
        <span
          aria-label="close-circle"
          className="anticon anticon-close-circle ant-input-clear-icon-hidden ant-input-clear-icon"
          onClick={[Function]}
          role="button"
          tabIndex={-1}
        >
          <svg
            aria-hidden="true"
            data-icon="close-circle"
            fill="currentColor"
            focusable="false"
            height="1em"
            viewBox="64 64 896 896"
            width="1em"
          >
            <path
              d="M512 64C264.6 64 64 264.6 64 512s200.6 448 448 448 448-200.6 448-448S759.4 64 512 64zm165.4 618.2l-66-.3L512 563.4l-99.3 118.4-66.1.3c-4.4 0-8-3.5-8-8 0-1.9.7-3.7 1.9-5.2l130.1-155L340.5 359a8.32 8.32 0 01-1.9-5.2c0-4.4 3.6-8 8-8l66.1.3L512 464.6l99.3-118.4 66-.3c4.4 0 8 3.5 8 8 0 1.9-.7 3.7-1.9 5.2L553.5 514l130 155c1.2 1.5 1.9 3.3 1.9 5.2 0 4.4-3.6 8-8 8z"
            />
          </svg>
        </span>
      </span>
    </span>
    <span
      className="ant-input-group-addon"
    >
      <button
        className="ant-btn ant-btn-primary ant-btn-lg ant-input-search-button"
        onClick={[Function]}
        onMouseDown={[Function]}
        type="button"
      >
        <span>
          Search
        </span>
      </button>
    </span>
  </span>
</span>
`;
